project_task_material_stock/models/project_task.py

```diff
@@ -177,10 +183,29 @@ def _prepare_stock_move(self):
 
     @api.multi
     def create_stock_move(self):
-        for line in self:
-            move_id = self.env['stock.move'].create(
-                line._prepare_stock_move())
-            line.stock_move_id = move_id.id
+        pick_type = self.env.ref('stock.picking_type_out')
+
+        if self:
```

Comment:
This if is not necessary. If you arrive here if because self is set.

Reply:
@cubells thank you, but when delete items, self is empty and crash all

Reply:
Done!, updated